_posts/2020-12-24-newpipe-0.20.7.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "NewPipe 0.20.7 released: YouTube Mix support, bug fixes and many improvements"
+short: "NewPipe 0.20.7 released"
+date: 2020-12-24 19:00:00
+categories: [pinned, release]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Merry Christmas, if you celebrate that, and almost Happy New Year, if you use a calendar at least close to the Gregorian one. We want to present you a little gift as a good way to start into 2021 AD: The next release of NewPipe. It comes with a lot of improvements, fixes, and even a big new feature. So let us dive into this release post!
+
+<!-- more -->
+
+## New in this version
+YouTube mixes are a feature now, and they are dropping in hot (and with a lot of bass). Opening a mix in previous versions resulted in the first video of the mix playing, nothing else. Now you see the full playlist and even a nice radio icon as an indicator. Enjoy listening to never-ending lists and thank [@Stypox](https://github.com/Stypox) as well as [@XiangRongLin](https://github.com/XiangRongLin) for this feature.
```

Comment:
I only reviewed, I'd give full credit to @XiangRongLin ;-)
If it's too late to suggest then nevermind
```suggestion
YouTube mixes are a feature now, and they are dropping in hot (and with a lot of bass). Opening a mix in previous versions resulted in the first video of the mix playing, nothing else. Now you see the full playlist and even a nice radio icon as an indicator. Enjoy listening to never-ending lists and thank [@XiangRongLin](https://github.com/XiangRongLin) for this feature.
```

Reply:
no thats fine but Im gonna still include you ;P cant escape.

```suggestion
YouTube mixes are a feature now, and they are dropping in hot (and with a lot of bass). Opening a mix in previous versions resulted in the first video of the mix playing, nothing else. Now you see the full playlist and even a nice radio icon as an indicator. Enjoy listening to never-ending lists and thank [@XiangRongLin](https://github.com/XiangRongLin) for this feature as well as [@Stypox](https://github.com/Stypox) for reviewing it.
```

how do you like this?

Reply:
Ok, fine :-D

Reply:
good.